### ISSUE TITLE
Check boxes when submitting issues from VS Code are missing accessible names when tabbing with VoiceOver

### ISSUE THREAD

**Issue by keford** [microsoft/vscode#140222]
Issue Type: <b>Bug</b>

1. With VoiceOver running, activate the submit issue option from the help menu.
2. Tab through the different fields and note what VoiceOver reads when you reach the check boxes.

Result:
The check boxes just say check box checked.  To know what you are checking you have to use the VoiceOver next item command.  This shiouldn't be required.

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:38:17.605Z)
OS version: Darwin arm64 21.3.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.16GB free)|
|Process Argv|--crash-reporter-id 93b64371-544c-441c-98de-23303e474b41|
|Screen Reader|yes|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263cf:30335440
vscorecescf:30384386
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
vscop804:30404766
vs360cf:30404996
vsrem710:30416614
py55gd98cf:30419166
vscexrecpromp3t1:30407762

```

</details>

**Comment by isidorn** [comment]
Yeah I can reproduce this. We are using native checkbox elements there
@TylerLeonhardt can we try putting `aria-label` on each native checkbox? That might help.
If that does not then we need to restructure the element a bit, so the parent element has the role checkbox, it has an aria-label with the details, and on space it actually checks the checkbox (similar to what we do for breakpoitns in breakpoints view)

**Comment by Ashwinipola26** [comment]
GitHubTags:#A11y_Visual Studio Code Client;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#Info and Relationships;#MAC;#AINotInScope;#VoiceOver;#BM-VisualStudioCodeClient-MAC-Aug2021;
